draft-ietf-ppm-dap.md

```diff
@@ -1741,18 +1741,8 @@ attack. For example, it may coerce a client into providing malicious input
 shares for aggregation or coerce an aggregator into diverting from the
 protocol specified (e.g., by divulging its input shares to the attacker).
 
-In the presence of this adversary, DAP aims to achieve the following high-level
-secure aggregation goals:
-
-1. Privacy. Clients trust that some aggregator is honest. That is, as long as at
-   least one aggregator executes the protocol faithfully, the parties learn nothing
-   beyond the aggregate result (i.e., the output of the aggregation function computed over
-   the honest measurements).
-1. Correctness. The collector trusts that the aggregators execute the protocol
-   correctly. That is, as long as the aggregators execute the protocol faithfully,
-   a malicious client can skew the aggregate result only by reporting
-   a false (untruthful) measurement. The result cannot be influenced in any
-   other way.
+In the presence of this adversary, DAP aims to achieve the privacy and
+robustness security goals described in {{!VDAF, Section 9}}.
```

Comment:
The section number is subject to change (if, for example, we add a new VDAF). Maybe refer to security considerations, just to make our lives later in the future.